components_new/query_handler/queryHandler.py

```
@@ -0,0 +1,41 @@
+import json
+from queryFunctions import *
+
+def handleQuery (message):
+    '''
+        Handle the query.
+    '''
+    if "queryType" not in message:
+        response = handleUnknownMessage(message= message)
```

Ruff (0.8.2)

`handleUnknownMessage` may be undefined, or defined from star imports

(F405)